database/src/migrations/20210819170006_add_soft_delete_timestamp_to_occurrence_submission.ts

```diff
@@ -0,0 +1,35 @@
+import * as Knex from 'knex';
+
+const DB_SCHEMA = process.env.DB_SCHEMA;
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+    set schema '${DB_SCHEMA}';
+    set search_path = ${DB_SCHEMA},public;
+
+    ALTER TABLE ${DB_SCHEMA}.occurrence_submission add column soft_delete_timestamp timestamptz(6);
```

Comment:
Not a major issue, bu I think just `delete_timestamp` is sufficient.  The soft part is implied by the fact that the record still exists, and was just marked deleted (rather than actually being gone from the DB)

Reply:
Also keeps it in line with the format of the existing `publish_timestamp`

Reply:
the soft delete terminology was what charlie had put in my ticket so I just went with it :)

Reply:
so if its ok, ill just leave it as is :)

Reply:
Sounds good